app/models/agents/freme_filter_agent.rb

```diff
@@ -7,9 +7,9 @@ class FremeFilterAgent < Agent
     default_schedule 'never'
 
     description <<-MD
-      The `FremeFilterAgent` allows to execute a certain filter against a RDF graph sent as post body or as value of the NIF input parameter. For more information and a list of available filters, see the [Simply FREME output using SPARQL filters](http://api.freme-project.eu/doc/0.5/knowledge-base/filtering.html) article.
+      The `FremeFilterAgent` allows to execute a certain filter against a RDF graph sent as post body or as value of the NIF input parameter. For more information and a list of available filters, see the [Simply FREME output using SPARQL filters](http://api.freme-project.eu/doc/0.6/knowledge-base/filtering.html) article.
```

Comment:
The new link seems to be http://api.freme-project.eu/doc/0.6/knowledge-base/freme-for-api-users/filtering.html